WORKSPACE

```diff
@@ -44,9 +44,16 @@ pip_install(
 # Load Kotlin
 load("@vaticle_dependencies//builder/kotlin:deps.bzl", kotlin_deps = "deps")
 kotlin_deps()
-load("@io_bazel_rules_kotlin//kotlin:kotlin.bzl", "kotlin_repositories", "kt_register_toolchains")
-kotlin_repositories()
+load("@io_bazel_rules_kotlin//kotlin:repositories.bzl", "kotlin_repositories", "kotlinc_version")
+kotlin_repositories(
+    compiler_release = kotlinc_version(
```

Comment:
This version (or some other version) is already pinned by the fact we pin our version of `rules_kotlin` - so let's not specify a version here.

Reply:
Unfortunately, we have to explicitly pin our version here as the default kotlinc version pinned by rules_kotlin is 1.7.10, which is not compatible with the very latest version of the compose compiler:
```
error: this version (1.3.2) of the Compose Compiler requires Kotlin version 1.7.20 but you appear to be using Kotlin version 1.7.10 which is not known to be compatible.  Please fix your configuration (or `suppressKotlinVersionCompatibilityCheck` but don't say I didn't warn you!).
```

Reply:
Hah, interesting, I guess we have to!